website/pages/en/new-chain-integration.mdx

```diff
@@ -1,75 +1,110 @@
 
----
+> NOTE: All integrations done by the StreamingFast team include maintenance for the Firehose replication protocol into the chain's codebase. StreamingFast tracks any changes and releases binaries when you change code and when StreamingFast changes code. This includes releasing Firehose/Substreams binaries for the protocol, maintaining Substreams modules for the block model of the chain, and releasing binaries for the blockchain node with instrumentation if need be.
 
-## Testing an EVM JSON-RPC
+#### Specific Firehose Instrumentation for EVM (`geth`) chains
 
-For Graph Node to be able to ingest data from an EVM chain, the RPC node must expose the following EVM JSON RPC methods:
+For the EVM chains, there exists a deeper level of data that can be achieved through the `geth` [live-tracer](https://github.com/ethereum/go-ethereum/releases/tag/v1.14.0), a collaboration between Go-Ethereum and StreamingFast, in building a high-throughput and rich transaction tracing system. The Live Tracer is the most comprehensive solution, resulting in [Extended](https://streamingfastio.medium.com/new-block-model-to-accelerate-chain-integration-9f65126e5425) block details. This enables new indexing paradigms, like pattern matching of events based on state changes, calls, parent call trees, or triggering of events based on changes to the actual variables in a smart contract.
 
-- `eth_getLogs`
-- `eth_call` \_(for historical blocks, with EIP-1898 - requires archive node):
-- `eth_getBlockByNumber`
-- `eth_getBlockByHash`
-- `net_version`
-- `eth_getTransactionReceipt`, in a JSON-RPC batch request
-- _`trace_filter`_ _(optionally required for Graph Node to support call handlers)_
+![Base block vs Extended block](/img/extended-vs-base-substreams-blocks.png)
+
+> NOTE: This improvement upon the Firehose requires chains make use of the EVM engine `geth version 1.13.0` and up.
 
-### Graph Node Configuration
+## EVM considerations - Difference between JSON-RPC & Firehose
 
-**Start by preparing your local environment**
+While the JSON-RPC and Firehose are both suitable for subgraphs, a Firehose is always required for developers wanting to build with [Substreams](https://substreams.streamingfast.io). Supporting Substreams allows developers to build [Substreams-powered subgraphs](/cookbook/substreams-powered-subgraphs) for the new chain, and has the potential to improve the performance of your subgraphs. Additionally, Firehose — as a drop-in replacement for the JSON-RPC extraction layer of `graph-node` — reduces by 90% the number of RPC calls required for general indexing.
+
+- All those `getLogs` calls and roundtrips get replaced by a single stream arriving into the heart of `graph-node` and by running filters/transformations (Substreams) on the in-memory blocks; a single block model for all subgraphs it processes.
```

Comment:
I don't understand this sentence. "and by running" is where I get confused – should it say "and running" or "by running"?

Reply:
No its additional to, like you can choose to do further filters and transformations - but I see confusion, I revised the sentence.